ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -474,12 +474,17 @@ trait MeteringStorageWriteBackend {
   ): Option[Offset]
 
   /** This method will return all transaction metering records between the from offset (exclusive)
-    * and the to offset (inclusive).
+    * and the to offset (inclusive).  It is called prior to aggregation.
     */
-  def transactionMetering(from: Offset, to: Offset)(
+  def selectTransactionMetering(from: Offset, to: Offset)(
```

Comment:
no need to change: maybe it would make the params cleaner if inclusive and exclusive is also in the name

Reply:
Can be a good technique.  In this case the method has scaladoc that explicitly explains the processing which is even better.